backend/application/syson-application/src/test/java/org/eclipse/syson/application/controllers/diagrams/general/view/GVEdgeCreationTests.java

```diff
@@ -153,13 +159,19 @@ public void createAddAsNestedEdge() {
             // Get the target with its description instead of its label: there are new two elements with the label
             // "action" on the diagram (one on the diagram itself, and one in the "actions" compartment of the part
             // element).
-            String targetId = diagramNavigator
-                    .nodeWithNodeDescriptionId(this.diagramDescriptionIdProvider.getNodeDescriptionId(this.descriptionNameGenerator.getNodeName(SysmlPackage.eINSTANCE.getActionUsage()))).getNode()
-                    .getId();
-            Edge newEdge = newDiagram.getEdges().get(0);
-            assertThat(newEdge).hasSourceId(sourceId);
-            assertThat(newEdge).hasTargetId(targetId);
-            assertThat(newEdge.getStyle()).hasSourceArrow(ArrowStyle.FillDiamond);
+            Optional<String> optionalTargetId = diagramNavigator.findAllNodes().stream()
```

Comment:
I made this test more resilient since my own test adds ActionUsage elements in the test model.